skit_calls/calls.py

```diff
@@ -9,7 +9,7 @@
 from skit_calls import constants as const
 from skit_calls.data import mutators, query
 from skit_calls.data.model import Turn
-
+from skit_calls import utils
 
```

Comment:
Nitpick: Importing only the needed functions and not the entire file should do

Reply:
Have changed this.